### PR TITLE
Fix StrContainsRector when strpos offset is set

### PR DESCRIPTION
**Pull request by ajgarlag**
Based on https://getrector.org/demo/e99068d4-3691-4653-ab58-24f20efa13bf

This is my first PR on this project. Thanks for your work!

### CODE REVIEW

File: rules/Php80/Rector/NotIdentical/StrContainsRector.php [hunk 1]
```diff
@@ -82,6 +85,14 @@ public function refactor(Node $node): ?Node
             return null;
         }
 
+        if (isset($funcCall->args[2])) {
+            if ($this->isName($funcCall->name, 'strpos') && $this->isPositiveInteger($funcCall->args[2]->value)) {
+                $subStrFuncCall = new FuncCall(new Name('substr'), [$funcCall->args[0], $funcCall->args[2]]);
+                $funcCall->args[0] = new Arg($subStrFuncCall);
```
Comment (samsonasik) [inline rules/Php80/Rector/NotIdentical/StrContainsRector.php]: You can use nodeFactory service for it:

```suggestion
                $funcCall->args[0] = new Arg($this->nodeFactory->createFuncCall('substr', [$funcCall->args[0], $funcCall->args[2]]));
```

**Review by samsonasik — APPROVED** [review]

File: rules/Php80/Rector/NotIdentical/StrContainsRector.php [hunk 2]
```diff
@@ -82,6 +85,13 @@ public function refactor(Node $node): ?Node
             return null;
         }
 
+        if (isset($funcCall->args[2])) {
+            if ($this->isName($funcCall->name, 'strpos') && $this->isPositiveInteger($funcCall->args[2]->value)) {
```
Comment (staabm) [inline rules/Php80/Rector/NotIdentical/StrContainsRector.php:89]: I guess the same is true for `stripos`?

Comment (ajgarlag) [reply rules/Php80/Rector/NotIdentical/StrContainsRector.php:89]: This rule only works with `strpos` y `strstr`

**Comment by TomasVotruba** [comment]
Thank you :+1:

**Pull request merged**